Clarify that `pad1` and `pad2` must only be filled with zero-value (0x00) bytes (#79)
creator-assertions/identity-assertion: docs/modules/ROOT/pages/index.adoc

* `referenced_assertions` contains a list of assertions that are referenced and signed by this _<<_credential_holder,credential holder>>_
* `signature` contains the raw byte stream of the _<<_credential_holder,credential holder>>’s_ signature
* `sig_type` defines the data type of signature

> * `pad1` and `pad2` are zero-value (`0x00`) byte strings used to fill space

Reviewer: ```suggestion
* `pad1` and `pad2` are zero-filled (binary `\0` values) byte strings used to fill space
```

> * `pad1` and `pad2` are zero-filled (binary `0x00` values) byte strings used to fill space

Author: Suggested reword: ... are byte strings filled with binary `0x00` values ...

> * `pad1` and `pad2` are zero-filled (binary `\0` values) byte strings used to fill space

Reviewer: ```suggestion
* `pad1` and `pad2` are zero-filled (binary `0x00` values) byte strings used to fill space
```

  "referenced_assertions": [1* $hashed-uri-map],
  "sig_type": tstr .size (1..max-tstr-length), ; A string identifying the data type of the signature field that follows
  "signature": bstr, ; byte string of the signature

>   "pad1": bstr, ; zero-value (`0x00`) byte string used for filling up space

Reviewer: ```suggestion
  "pad1": bstr, ; zero-filled (binary `\0` values) byte string used for filling up space
```

>   ? "pad2": bstr, ; optional zero-value (`0x00`) byte string used for filling up space

Reviewer: ```suggestion
  ? "pad2": bstr, ; optional zero-filled (binary `\0` values) byte string used for filling up space
```
